ui/apps/platform/src/Containers/Collections/hooks/useEmbeddedCollections.ts

```diff
@@ -64,7 +64,8 @@ function fetchDetachedCollections(
         );
         const detached = aggregateResult.concat(newDetached);
         const lastResponseSize = collections.length;
-        const shouldFetchMore = lastResponseSize > 0 && detached.length < minimumUpdateSize;
+        const shouldFetchMore =
```

Comment:
With this change. we only automatically fetch more for the available collection list if the following conditions are met:
- The `lastResponseSize` of the previous fetch was equal to the size of the pages we are fetching (if it is less, then no more data exists on the server)
- The amount of collections we are adding to the available list is less than the `minimumUpdateSize`, which is half the page size. This handles situations where we fetch a page of data, but > half of those collections already exist in the attached list.

Previously, with `lastResponseSize > 0`, we would get one redundant fetch when the last page was returned and we were below the minimum update size. This happened frequently when there was very little data in the system and you were viewing a collection that has any embedded collections.